### ISSUE TITLE
No minimize or maximize windows option

### ISSUE THREAD

**Issue by scotthibbs** [w7sst/MorseRunner#294]
## Summary

When testing the program, I found myself accidently closing the program to write an email or look at github and discovered the reason I am doing this as there is only a red X close button at the top of the form. I'm clicking this out of habit to remove the program from the desktop when I should click away. The user may also want to maximize the program. 

## Motivation

 This is a normal function of a modern windows program.

## Detailed Description

The normal maximize and minimize buttons at the top of the form of the program. 
## Can you help?

Please let us know if you are available to help. (replace '[ ]' with '[x]' to affirm)
- [x] Yes, I'm available to discuss possible solutions and alternatives to this problem.
- [x] Yes, I'd like to review and provide feedback on the proposed solution.
- [x] Yes, I'm available to help test a solution to this problem.

## Tasks
- [x] Coding and integration
- [ ] Validation

**Comment by w7sst** [comment]
I discovered it is easy to add either Minimize or Maximize buttons to the application window/form. I tried both. The Minimize button works great. However, the Maximize button maximizes the application window, but the form is not designed to be maximize yet (the log window becomes very large; the other application areas do not resize, nor should they).

Anyway, adding the Minimize is easy to to.

@scotthibbs
